src/features/island/bumpkin/components/DynamicMiniNFT.tsx

```diff
@@ -43,6 +43,7 @@ import snowman from "assets/npc-layers/snowman-onesie.png";
 import sharkOnesie from "assets/npc-layers/shark-onesie.png";
 import reindeerSuit from "assets/npc-layers/reindeer_suit.png";
 import reindeerAntlers from "assets/npc-layers/reindeer_antlers.png";
+import lionDanceMask from "assets/npc-layers/lion_dance_mask.png";
```

Comment:
feel free to download the original spritesheet on my now closed PR https://github.com/sunflower-land/sunflower-land/pull/2096

Reply:
Sure thing! It will be checked in and pushed to this PR in a few minutes.